sections/iana.include

```diff
@@ -37,8 +37,8 @@
       :: The <code>charset</code> parameter may be provided to specify the
           <a>document's character encoding</a>, overriding any
           [=character encoding declarations=] in the document other than a Byte Order Mark (BOM).
-          The parameter's value must be one of the <a lt="character encoding">labels</a> of the <a>character encoding</a>
-          used to serialize the file. [[!ENCODING]]
+          The parameter's value must be an <a>ASCII case-insensitive</a> match for the string
+          "<code>utf-8</code>". [[!ENCODING]]
```

Comment:
If we just enforce a string, there is no normative dependency on the encoding spec here.